addons/mail/static/src/components/message/message.xml

```diff
@@ -19,10 +19,11 @@
             }" role="group" t-att-aria-label="messageView and messageView.message.messageTypeText" t-on-click="_onClick" t-on-mouseenter="state.isHovered = true" t-on-mouseleave="state.isHovered = false" t-att-data-message-local-id="messageView and messageView.message.localId"
         >
             <t t-if="messageView" name="rootCondition">
-                <MessageInReplyToView t-if="messageView.messageInReplyToView" messageInReplyToViewLocalId="messageView.messageInReplyToView.localId"/>
+                <MessageInReplyToView t-if="messageView.messageInReplyToView and threadView and !threadView.threadViewer.chatWindow" messageInReplyToViewLocalId="messageView.messageInReplyToView.localId"/>
                 <div t-if="isActive and messageView.messageActionList" class="o_Message_actionListContainer pl-5 pr-3" t-att-class="{ 'o-squashed': messageView.isSquashed }">
                     <MessageActionList messageActionListLocalId="messageView.messageActionList.localId"/>
                 </div>
+                <MessageInReplyToView t-if="messageView.messageInReplyToView and threadView and threadView.threadViewer.chatWindow" messageInReplyToViewLocalId="messageView.messageInReplyToView.localId"/>
```

Comment:
This diff looks useless. In chat window, `o_Message_actionListContainer` is absolute positioning, so having `MessageInReplyToView` before or after it doesn't matter